media-sound/drumkv1/drumkv1-9999.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit fdo-mime
+
+DESCRIPTION="An old-school all-digital drum-kit sampler synthesizer with stereo fx"
+HOMEPAGE="http://drumkv1.sourceforge.net/"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3 autotools
+	EGIT_REPO_URI="https://github.com/rncbc/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://download.sourceforge.net/${PN}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+LICENSE="GPL-2"
+SLOT="0"
+RESTRICT="mirror"
+
+IUSE="debug -qt4 +standalone alsa +jackmidi jacksession lv2 osc"
+REQUIRED_USE="
+	|| ( standalone lv2 )
+	standalone? ( || ( alsa jackmidi ) )
+	jackmidi? ( standalone )
+	jacksession? ( standalone )"
+
+RDEPEND="
+	qt4? (
+		dev-qt/qtcore:4
+		dev-qt/qtgui:4
+	)
+	!qt4? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtxml:5
+	)
+	media-libs/libsndfile
+	standalone? ( virtual/jack )
+	alsa? ( media-libs/alsa-lib )
+	lv2? ( media-libs/lv2 )
+	osc? ( media-libs/liblo )
+"
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	if [[ ${PV} == *9999 ]]; then
+		eautoreconf
+	fi
+
+	# Disable stripping
+	echo "QMAKE_STRIP=" >> src/src_core.pri.in
+	echo "QMAKE_STRIP=" >> src/src_jack.pri.in
+
+	local -a myeconfargs=(
+		$(use_enable debug)
```

Comment:
All packages I pushed into upstream people told me to use `usex` instead of `use_enable` because it is better to read. And I must agree with them. The syntax of `use_enable` `use_with` must always be looked up, and these commands safe you just 3 minus symbols and equal symbol.